hubble/hubble.go

```diff
@@ -78,11 +84,8 @@ type Parameters struct {
 	UI               bool
 	UIPortForward    int
 	Writer           io.Writer
-	Context          string // Only for 'kubectl' pass-through commands
-}
-
-func (p *Parameters) Log(format string, a ...interface{}) {
-	fmt.Fprintf(p.Writer, format+"\n", a...)
```

Comment:
I think this part of the patch is not intended, I'm unable to build cilium-cli:

```
~/go/src/github.com/cilium/cilium-cli % make
CGO_ENABLED=0 go build  \
	-ldflags "-w -s \
	-X 'github.com/cilium/cilium-cli/internal/cli/cmd.GitBranch=pr/169' \
	-X 'github.com/cilium/cilium-cli/internal/cli/cmd.GitHash=5958fe2' \
	-X 'github.com/cilium/cilium-cli/internal/cli/cmd.Version=0.8.0-dev'" \
	-o cilium \
	./cmd/cilium
# github.com/cilium/cilium-cli/hubble
hubble/relay.go:409:6: p.Context undefined (type *Parameters has no field or method Context)
hubble/relay.go:410:40: p.Context undefined (type *Parameters has no field or method Context)
hubble/ui.go:326:6: p.Context undefined (type *Parameters has no field or method Context)
hubble/ui.go:327:40: p.Context undefined (type *Parameters has no field or method Context)
hubble/ui.go:342:5: p.Log undefined (type *Parameters has no field or method Log)
hubble/ui.go:343:5: p.Log undefined (type *Parameters has no field or method Log)
make: *** [Makefile:18: cilium] Error 2
```

Reply:
Ok so the only thing that's to be done is to remove this chunk of code?

Reply:
Technically, since the patch remove the `Context` field and the `Log` function from `Parameters`, what is required is to get them back in.

If you meant removing it from the diff of this PR, then the answer is yes.

Reply:
So do I add the context field and the log function or do I remove that chunk of code? I am sorry but I didn't quiet understand.